src/route.shared.ts

```diff
@@ -263,10 +261,10 @@ async function loadPageRoutes(): Promise<Record<PageId, PageRoute>> {
         `The default export of ${filePath} should be a string or a function.`
       )
       const pageRoute = fileExports.default
-      const pageId = computePageId(filePath)
+      const id = computePageId(filePath)
       const pageRouteFile = filePath
 
-      pageRoutes[pageId] = { pageRoute, pageRouteFile }
+      pageRoutes[id] = { pageRoute, pageRouteFile, id }
```

Comment:
Let's keep it to `pageId`; having consistent naming makes reading source code considerably easier.